pkgs/by-name/le/leftwm-config/0001-rm-unstable-is-some-with-feature.patch

```diff
@@ -0,0 +1,93 @@
+diff --git a/src/main.rs b/src/main.rs
```

Comment:
I dont see any `patches = [ ./0001-rm-unstable-is-some-with-feature.patch ];`. Is the patch used?

Reply:
Oh! It was part of the original commits I've based my work upon. The patch seemed needed at the time, but I disabled it whilst keeping it as it seemed to run without, then lost track of it. Will be removed!